bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/container.jsx

```diff
@@ -3,20 +3,30 @@ import { withTracker } from 'meteor/react-meteor-data';
 import Breakouts from '/imports/api/breakouts';
 import Meetings from '/imports/api/meetings';
 import Auth from '/imports/ui/services/auth';
-import mapUser from '/imports/ui/services/user/mapUser';
 import UserListItem from './component';
+import UserListService from '/imports/ui/components/user-list/service';
+import AppService from '/imports/ui/components/app/service';
 
 const UserListItemContainer = props => <UserListItem {...props} />;
 
 export default withTracker(({ user }) => {
   const findUserInBreakout = Breakouts.findOne({ 'joinedUsers.userId': new RegExp(`^${user.userId}`) });
   const breakoutSequence = (findUserInBreakout || {}).sequence;
-  const Meeting = Meetings.findOne({ MeetingId: Auth.meetingID }, { fields: { meetingProp: 1 } });
+  const Meeting = Meetings.findOne({ meetingId: Auth.meetingID },
+    { fields: { lockSettingsProps: 1 } });
+
+  const isMe = intId => intId === Auth.userID;
```

Comment:
This function should be defined outside the tracker. Or instead just do `const isMe = user.userId === Auth.userId` and then work with the boolean in the user-list-item.